forms-web-app/src/lib/views.js

```diff
@@ -33,4 +33,6 @@ const VIEW = {
 
 module.exports = {
   VIEW,
+  WHO_ARE_YOU: VIEW.APPELLANT_SUBMISSION.WHO_ARE_YOU,
+  APPLICATION_NUMBER: VIEW.APPELLANT_SUBMISSION.APPLICATION_NUMBER,
```

Comment:
feels odd to have `VIEW` and then the next two using `VIEW`.

Perhaps `VIEW` doesn't belong here anymore? idk... not something for this PR I don't think, just a thought.

Reply:
Yes it was legacy for test. Will clean it